eng/testing/debug-dump-template.md

```diff
@@ -58,8 +61,8 @@ Open WinDbg and open the dump with `File>Open Dump`.
 ```
 
 ```
-!setclrpath %WOUTDIR%\shared\Microsoft.NETCore.App\6.0.0
-.sympath+ %WOUTDIR%\shared\Microsoft.NETCore.App\6.0.0
+!setclrpath %WOUTDIR%\shared\Microsoft.NETCore.App\7.0.0
```

Comment:
NIT: maybe the version we could make it a placeholder as well and pass it down as a parameter to the Python script since we set the parameters on the project that sends to helix and that has access to the current version we are targeting.

Reply:
Yup, that makes perfect sense.

@safern, would it be good if we replace the python script with an MSBuild target? We can then use `$(NetCoreAppCurrentVersion)` string from `:(top)/Directory.Build.props`. We are generating shell scripts in targets in other places, so it won't be the first time. and one less python file.. :)

Reply:
This file gets populated in Helix, which guarantees python in the machine/container. We don't deploy an SDK always, and the job name is not given until after we submit, so we can't pack it in the work item. I'd say python in this part of the stack is ok.

Reply:
Yea the reason why we use Python is because the generator is invoked on helix after the test runs and helix is Python driven, we have no guarantee than a .NET SDK will be present because we just populate a runtime to run the tests.

Reply:
My 2c, we'll essentially be updating this once a year, it may not be worth bothering with automating the version number. In the rare case we're looking at a crash in servicing, we'll figure it out..

Reply:
Yup, we can still extract it as a placeholder and hardcode it in python script (one place) with some
discoverable / `grep`'able name (like `NetCoreAppCurrentVersion=7.0`) so it doesn't fall under the radar in the annual
"Update Version" PRs. :)

> This file gets populated in Helix, which guarantees python in the machine/container.

Ah, right. I had a doubt something like this was in play which is why I asked-first rather than code-first. 😉

Reply:
Something like https://github.com/am11/runtime/commit/7a7f37f991cdad61ba0ab796768e9e54a447a3ec?

Reply:
I would instead use `ProductVersion` which is a full semver representation. https://github.com/dotnet/runtime/blob/acb00500a08319e3b97af3bfa3c029e7620defdb/eng/Versions.props#L4

Reply:
Ah, but we would need to pass it all the way from helix.yml and parameterize all callsites. 😕

Reply:
I think `Versions.props` is imported in the context of `sendtohelix.proj` and `sendtohelixhelp.proj`